content/blog/doomsday-rule/index.md

```diff
@@ -0,0 +1,660 @@
+---
+{
+    title: "Doomsday Rule",
+    description: 'In this blog I talk about the Doomsday Rule, how it works, how to put it into code then how to make a program that tests you.',
+    published: '2019-06-29T22:12:03.284Z',
+    authors: ['SkyHawk_0'],
+    tags: ['Python', 'Maths', 'Doomsday rule'],
+    attached: [],
+    license: 'cc-by-4'
+}
+---
+
+# What is the doomsday rule?
+
+Before I get to how my program works, I should probably explain what the doomsday rule is. The doomsday rule is an algorithm that determines the day of the week for a given date in the Gregorian calendar, e.g. the 18th of March 1898 is a Friday. I can't think when you would use this in everyday life but it's a cool party trick to show off with.
```

Comment:
```suggestion
Before I get to how my program works, I should probably explain what the doomsday rule is. The doomsday rule is an algorithm that determines the day of the week for a given date in the Gregorian calendar. For example, without looking at a calendar, I know that the 18th of March, 1898 is a Friday. I can't think when you would use this in everyday life, but it's a cool party trick to show off with.
```

Minor grammar change (adding commas), which lead to a minor rewrite to help with flow.